### PR TITLE
Fix recovery bug:true every time after reversed

### PR DESCRIPTION
**Pull request by shuaiLiWang**
## What does this implement/fix? Explain your changes.
修复恢复出错bug

## Does this close any currently open issues?  
If it fixes a bug or resolves a feature request, be sure to link to that issue.
https://github.com/OneKeyHQ/TaskHub/issues/1371

## Pull request type
 
_Put an `x` in the boxes that apply_
- [x] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, renaming)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] Documentation content changes
- [ ] Other (please describe): 

## Where has this been tested?
…

## Any other comments?
…

### CODE REVIEW

**Comment by taimanhui** [comment]
找到错误原因了，在#730 （commit  9b05f67f5a11cdeee14caac4c65e26a08940c1d4）这个里面，我的错，没注意到reversed是返回一个iterator...

File: electrum_gui/android/console.py [hunk 1]
```diff
@@ -3282,7 +3282,7 @@ def filter_wallet(self):
                 fiat_str = self.daemon.fx.ccy_amount_str(main_coin_balance * main_coin_price, True)
                 balance = f"{main_coin_balance} ({fiat_str} {self.ccy})"
             else:
-                history = reversed(wallet.get_history())
+                history = wallet.get_history()
                 if not history:
                     continue
```
Comment (taimanhui) [inline electrum_gui/android/console.py:3287]: 直接改成
```python
if not bool(wallet.get_history()):
    continue
```
就好，这里其实我们并不关注历史本身...

**Review by taimanhui — APPROVED** [review]
@danielsocials @huazhouwang 需要多一个approve让这个修复尽快加入

File: electrum_gui/android/console.py (hunk 1 above)
Comment (shuaiLiWang) [reply electrum_gui/android/console.py]: ack

**Review by danielsocials — APPROVED** [review]

**Pull request merged**